fcl/dunefd/gen/background/prodbackground_radiological_decay0_dune10kt_1x2x6.fcl

```diff
@@ -140,7 +48,12 @@ outputs:
    }
 }
 
-services.Geometry: @local::dune10kt_1x2x6_geo
+services:
+{
+    @table::services
+    @table::dunefd_1x2x6_services
+}
+
 services.message.destinations.LogStandardOut.categories.BackTracker.limit: 0
 services.message.destinations.LogStandardError.categories.BackTracker.limit: 0
```

Comment:
Not sure, tbh. This was already somewhere but it could be a reminiscent of a legacy file, where gen and g4 were run together. If you think it is fine to remove, I will remove it.

Reply:
Yeah let's gut these.  We should be using the default logging in production files IMO (and these lines don't do anything because its backtracker related anyway)

Reply:
Done. Should we keep these?
```
services.message.destinations.LogStandardOut.categories.BaseRadioGen.limit: 1
services.message.destinations.LogStandardError.categories.BaseRadioGen.limit: 1

```